config/default.json

```diff
@@ -31,8 +30,9 @@
         "enabled": true,
         "tests": {
             "homepage": {
+                "cookieName": "blf-ab-1-50",
```

Comment:
It's confusing but the reason this cooke's name was defined in the `cookies` array is because `generate-cloudfront-config.js` reads this array out and adds the cookies inside it to each Cloudfront route (eg. to allow those cookies through the load balancer). Either we need to modify that function to look in this location too, or leave the name in the `cookies` array.

Reply:
Ah gotcha 👌